### ISSUE TITLE
Google Fonts' version of this font is blurry

### ISSUE THREAD

**Issue by OscarAbraham** [JetBrains/JetBrainsMono#600]
This is on Windows. [The font from Google Fonts' repository](https://fonts.google.com/specimen/JetBrains+Mono) looks like this:
![Screenshot 2023-01-30 233200](https://user-images.githubusercontent.com/1027799/215674616-55558a0e-38e3-4dce-a915-f8a0a68ff7f0.png)
It's specially noticeable in the "e" and "t" characters.  For comparison, here's how that looks in [JetBrainsMono's web page](https://www.jetbrains.com/lp/mono/):
![Screenshot 2023-01-30 233238](https://user-images.githubusercontent.com/1027799/215675121-c582b257-5fe8-48b4-bd8e-51179c84128e.png)

I know that web page uses an older version of the font, but I've tried with the newest woff2 from the releases in this repository and it also looks much better than Google Fonts' version. My guess is that it has something to do with Google Fonts using a variable font; I've noticed that several other variable fonts in Google Fonts look blurry too.

**Comment by icarusgk** [comment]
Yeah, I think it might have to do with JetBrains Mono being variable and some issues with Windows too like Subpixel rendering, although I'm not completely convinced since the latest release of the font appears to look good on your system.
